codegen/initializer.ts

```diff
@@ -40,21 +40,30 @@ export type NDeepPartial<T> =
 interface DeepPartialSet<ItemType> extends Set<NDeepPartial<ItemType>> { }
 interface DeepPartialMap<KeyType, ValueType> extends Map<NDeepPartial<KeyType>, NDeepPartial<ValueType>> { }
 
+const empty = new Set<string>();
 
-function applyTo(source: any, target: any, cache = new Set<any>()) {
+function applyTo(source: any, target: any, filter: Set<string>, cache = new Set<any>(), ) {
```

Comment:
`excluded` could be a more accurate name since `filter` usually implies a predicate.  Should the type be `Set<string>` or `Set<any>`?

Reply:
Yeah, I can see `exclusions`

`Set<string>`, since it's just property names.  (and specifically not symbols or indexes)